materialize-css/materialize-css.d.ts

```diff
@@ -0,0 +1,378 @@
+// Type definitions for materialize-css
+// Project: http://materializecss.com/
+// Definitions by: Erik Lieben <https://github.com/eriklieben>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+
+/// <reference path="../jquery/jquery.d.ts" />
+
+declare module Materialize {
+
+    /**
+     * The collapsible options
+     */
+    interface ICollapsibleOptions {
```

Comment:
http://www.typescriptlang.org/Handbook#writing-dts-files

> Naming Conventions
In general, do not prefix interfaces with I (e.g. IColor). Because the concept of an interface in TypeScript is much more broad than in C# or Java, the IFoo naming convention is not broadly useful.